examples/fhi-aims/h2o-geop-fixatoms/ase_comparison/opt.log

```diff
@@ -0,0 +1,40 @@
+      Step     Time          Energy         fmax
```

Comment:
This and the next xyz files should be in a "regression test" type of folder. Not really in the examples folder. Examples are meant to tell people how to run something.